packages/bpk-component-carousel/src/BpkCarouselImage.module.scss

```diff
@@ -0,0 +1,38 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.bpk-carousel-image {
+  display: inline-flex;
+  width: 100%;
+  min-width: 0;
+  height: 100%;
+  min-height: 0;
+  justify-content: center;
+  scroll-snap-align: start;
+  scroll-snap-stop: always;
```

Comment:
Can we check these in Safari as according to https://caniuse.com/?search=scroll-snap-stop it seems it isn't supported in Safari 14 which is our current lowest version supported

Reply:
Yeah, I tried to remove `scroll-snap-stop` setting, the carousel doesn't break, could we leave it here to cover most of users to have a good experience?